Sources/SwiftCrossUI/Backend/AppBackend.swift

```diff
@@ -603,6 +604,91 @@ public protocol AppBackend: Sendable {
+    /// sheet gets dismissed. The sheet will be closed once onDismiss gets called
+    ///
+    /// Must only get called once for any given sheet.
+    ///
+    /// If `window` is `nil`, the backend can either make the sheet a whole
+    /// app modal, a standalone window, or a modal for a window of its choosing.
+    func showSheet(
+        _ sheet: Sheet,
+        window: Window?
+    )
+
+    /// Dismisses a sheet programmatically.
+    /// Gets used by the SCUI sheet implementation to close a sheet.
+    func dismissSheet(_ sheet: Sheet, window: Window?)
+
+    /// Sets the corner radius for a sheet presentation.
+    ///
+    /// This method is called when the sheet content has a `presentationCornerRadius` modifier
+    /// applied at its top level. The corner radius affects the sheet's presentation container,
+    /// not the content itself.
+    ///
+    /// - Parameters:
+    ///   - sheet: The sheet to apply the corner radius to.
+    ///   - radius: The corner radius
+    func setPresentationCornerRadius(of sheet: Sheet, to radius: Double)
+
+    /// Sets the available detents (heights) for a sheet presentation.
+    ///
+    /// This method is called when the sheet content has a `presentationDetents` modifier
+    /// applied at its top level. Detents allow users to resize the sheet to predefined heights.
+    ///
+    /// - Parameters:
+    ///   - sheet: The sheet to apply the detents to.
+    ///   - detents: An array of detents that the sheet can be resized to.
+    func setPresentationDetents(of sheet: Sheet, to detents: [PresentationDetent])
+
+    /// Sets the visibility for a sheet presentation.
+    ///
+    /// This method is called when the sheet content has a `presentationDragIndicatorVisibility`
+    /// modifier applied at its top level.
+    ///
+    /// - Parameters:
+    ///   - sheet: The sheet to apply the detents to.
+    ///   - visibility: visibility of the drag indicator (visible or hidden)
+    func setPresentationDragIndicatorVisibility(
+        of sheet: Sheet,
+        to visibility: PresentationDragIndicatorVisibility
+    )
+
+    /// Sets the background color for a sheet presentation.
+    ///
+    /// This method is called when the sheet content has a `presentationBackground`
+    /// modifier applied at its top level.
+    ///
+    /// - Parameters:
+    ///   - sheet: The sheet to apply the detents to.
+    ///   - color: rgba background color
+    func setPresentationBackground(of sheet: Sheet, to color: Color)
+
+    /// Sets the interactive dismissablility of a sheet.
+    /// when disabled the sheet can only be closed programmatically,
+    /// not through users swiping, escape keys or similar.
+    ///
+    /// This method is called when the sheet content has a `interactiveDismissDisabled`
+    /// modifier applied at its top level.
+    ///
+    /// - Parameters:
+    ///   - sheet: The sheet to apply the detents to.
+    ///   - disabled: wether its disabled
```

Comment:
`The sheet to apply the interactive dismissibility to.`

`Whether interactive dismissing is disabled.`